ostree_upload_server/bundle_importer.py

```diff
@@ -0,0 +1,65 @@
+import inspect
+import logging
+import os
+
+import magic as file_magic
```

Comment:
Can you add a comment about the magic module used here? In particular, that the API here corresponds to the file-magic pypi module and not the python-magic pypi module? I'd like something to prod me when I update the server to 18.04 and suddenly the python-magic package provides a new API.